src/ips_options/ips_sd_pattern.cc

```diff
@@ -326,9 +329,12 @@ class SdPatternModule : public Module
 
 private:
     SdPatternConfig config;
+
+    static void scratch_setup(SnortConfig* sc);
+    static void scratch_cleanup(SnortConfig* sc);
 };
 
-bool SdPatternModule::begin(const char*, int, SnortConfig*)
+bool SdPatternModule::begin(const char*, int, SnortConfig* sc)
```

Comment:
Was something supposed to happen here with sc?  If not please delete.

ips_sd_pattern.cc:337:60: warning: unused parameter 'sc' [-Wunused-parameter]